src/parse.rs

```diff
@@ -25,19 +35,33 @@ pub fn get_file_path_from_file_meta_line(line: &str) -> String {
     } else {
         match line.split(' ').nth(1) {
             Some("/dev/null") => "/dev/null",
-            Some(path) => &path[2..],
+            Some(path) => {
+                if remove_prefix {
+                    &path[2..]
+                } else {
+                    path
+                }
+            }
             _ => "",
         }
         .to_string()
     }
 }
 
-pub fn get_file_change_description_from_file_paths(minus_file: &str, plus_file: &str) -> String {
-    match (minus_file, plus_file) {
-        (minus_file, plus_file) if minus_file == plus_file => minus_file.to_string(),
-        (minus_file, "/dev/null") => format!("deleted: {}", minus_file),
-        ("/dev/null", plus_file) => format!("added: {}", plus_file),
-        (minus_file, plus_file) => format!("renamed: {} ⟶   {}", minus_file, plus_file),
+pub fn get_file_change_description_from_file_paths(
+    minus_file: &str,
+    plus_file: &str,
+    comparing: bool,
+) -> String {
+    if comparing {
+        format!("comparing: {} ⟶   {}", minus_file, plus_file)
```

Comment:
Thanks, I like your style decision here to make this harmonious with the git diff "renamed" output. (I just have that concern above about not including the date timestamps).